plugins/hls-hlint-plugin/src/Ide/Plugin/Hlint.hs

```diff
@@ -609,7 +601,7 @@ applyHint recorder ide nfp mhint =
                 (anns', modu') <-
                     ExceptT $ mapM (uncurry Refact.applyFixities)
                             $ postParseTransform (Right (anns, [], dflags, modu)) rigidLayout
-                liftIO $ (Right <$> withRuntimeLibdir (Refact.applyRefactorings' position commands anns' modu'))
+                liftIO $ (Right <$> Refact.applyRefactorings' position commands anns' modu')
```

Comment:
This is curious, this code-path is definitely not exercised for GHC >= 9.2, since the API in apply-refact has changed in 0.11.0. And in apply-refact <= 0.10.0, I don't think we need the hack `withRuntimeLibdir` because the libdir is only needed for parsing and we pass in a module already.